ehr/src/org/labkey/ehr/utils/TriggerScriptHelper.java

```diff
@@ -1142,7 +1146,7 @@ private String checkOtherDrawsQuantity(String id, Date date, String rowObjectId,
                             foundRow = true;
                         }
                     }
-                    BloodInfo bloodsIntransc = new BloodInfo(objectId, ConvertHelper.convert(map.get("date"), Date.class), ConvertHelper.convert(map.get("quantity"), Double.class));
+                    BloodInfo bloodsIntransc = new BloodInfo(objectId, ConvertHelper.convert(map.get("date"), Date.class), ConvertHelper.convert(map.get("quantity"), Double.class), true);
```

Comment:
Is 'inTransaction' ever not true? What's it needed for?

Reply:
it is not true if it's strictly coming from the database, i didn't want to report the warning if it was a record that was already in the database, it seemed like those were still coming through

Reply:
yeah, here: https://github.com/LabKey/ehrModules/blob/18e14d2ebf8a2d05d8586b6dc9f146c20f579791/ehr/src/org/labkey/ehr/utils/TriggerScriptHelper.java#L1201

Reply:
Alright - I think I get what you're doing. This is another case where I think it's important to leave documentation within the code on the purpose of these toggles. Can you add something around this, maybe in the BloodInfo class? This doesnt need to be formal javadoc, but some explanatory comments are a good idea

Reply:
I added a comment to the new class member